### PR TITLE
Archana_Assignment_19_70min

### PR DESCRIPTION
**Pull request by ArchanaNJun20**
PrimeNumberOpration_40min
image
FirstDifferIndex_8min
image
AllDifferIndex_8min
image
MaxDiffer_8min
image
AlldDifferString_8min
image

### CODE REVIEW

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
Seriously Archana, All method name is the same, So why you have created so many methods just call only one methods. Nobody going to check your code if you have written like this.

File: src/archanaNJun20/PrimeNumberOpration.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package archanaNJun20;
+
+import java.util.Scanner;
+
+public class PrimeNumberOpration {
+	void primeOperation(){
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/PrimeNumberOpration.java]: make this method parameterized.

Comment (ArchanaNJun20) [reply src/archanaNJun20/PrimeNumberOpration.java]: The code is updated based on the given comments.
method names changed, from next time onwards  I will take care while writing.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Now cod looks perfect, Thank you for considering all the changes. Good job.

**Pull request merged**